src/algebra/ordered_field.lean

```diff
@@ -104,174 +130,75 @@ div_mul_cancel b (ne_of_lt hc) ▸ mul_le_mul_of_nonpos_right h (le_of_lt hc)
 lemma div_le_of_mul_le_of_neg (hc : c < 0) (h : a * c ≤ b) : b / c ≤ a :=
 calc
    a   = a * c * (1 / c) : mul_mul_div a (ne_of_lt hc)
-    ... ≥ b * (1 / c)     : mul_le_mul_of_nonpos_right h (le_of_lt (one_div_neg_of_neg hc))
+    ... ≥ b * (1 / c)     : mul_le_mul_of_nonpos_right h (le_of_lt (one_div_neg.2 hc))
     ... = b / c           : eq.symm $ div_eq_mul_one_div b c
 
-lemma mul_lt_of_gt_div_of_neg (hc : c < 0) (h : a > b / c) : a * c < b :=
+lemma mul_lt_of_div_lt_of_neg (hc : c < 0) (h : b / c < a) : a * c < b :=
 div_mul_cancel b (ne_of_lt hc) ▸ mul_lt_mul_of_neg_right h hc
 
-lemma div_lt_of_mul_lt_of_pos (hc : c > 0) (h : b < a * c) : b / c < a :=
+lemma div_lt_of_mul_lt_of_pos (hc : 0 < c) (h : b < a * c) : b / c < a :=
 calc
    a   = a * c * (1 / c) : mul_mul_div a (ne_of_gt hc)
-   ... > b * (1 / c)     : mul_lt_mul_of_pos_right h (one_div_pos_of_pos hc)
+   ... > b * (1 / c)     : mul_lt_mul_of_pos_right h (one_div_pos.2 hc)
    ... = b / c           : eq.symm $ div_eq_mul_one_div b c
 
 lemma div_lt_of_mul_gt_of_neg (hc : c < 0) (h : a * c < b) : b / c < a :=
```

Comment:
here again the name of the lemma is strange (in particular, `mul_gt` should be `mul_lt`)